### ISSUE TITLE
Update `admin` to be a standalone app

### ISSUE THREAD

**Issue by dianabarsan** [medic/cht-core#6616]
**Describe the issue**
Currently, the admin app requires a significantly large number of webapp files (services, filters, directives, redux actions/reducers/selectors). 
This becomes a problem with the migration to Angular 10, where the files imported from webapp will, most likely, be unusable by admin. 

**Describe the improvement you'd like**
"Duplicate" all files that admin app requires from the webapp folder, preserving git line history as much as possible. (this includes their tests)

**Describe alternatives you've considered**
We could postpone this migration under the assumption that we may be able to use post-angular-10-migration webapp files as we do now, but it's a risk.

**Comment by dianabarsan** [comment]
This is ready for AT on `6616-standalone-admin`

No changes were made to actual functionality, so just check the admin app still works.

**Comment by dianabarsan** [comment]
I've just merged this to unblock the Angular 10 migration. It was important to start from a place where we can safely move and rename files from webapp without affecting the admin app.

**Comment by MaxDiz** [comment]
Sounds like this issue should be close, correct?

**Comment by garethbowen** [comment]
@MaxDiz We're leaving it open so it can be ATed post merge.

**Comment by garethbowen** [comment]
@newtewt This one is ready for AT. It's already been merged to `master` so you can just test there.

**Comment by newtewt** [comment]
Since this is merged. I'm going to close. I've gone through the tabs and tested that the functionality is still working in master.  Found a couple issues but none of them are because of this change. Opened a new issue for default lang and commented on another for exporting people.
